src/app/rpmostree-dbus-helpers.c

```diff
@@ -68,27 +66,18 @@ get_connection_for_path (gchar *sysroot,
       if (sysroot != NULL)
         g_warning ("RPMOSTREE_USE_SESSION_BUS set, ignoring --sysroot=%s", sysroot);
 
-      connection = g_bus_get_sync (G_BUS_TYPE_SESSION, cancellable, error);
-      goto out;
+      return g_bus_get_sync (G_BUS_TYPE_SESSION, cancellable, error);
     }
 
   if (sysroot == NULL)
     sysroot = "/";
 
   if (g_strcmp0 ("/", sysroot) == 0 && force_peer == FALSE)
-    {
-      connection = g_bus_get_sync (G_BUS_TYPE_SYSTEM, cancellable, error);
-      goto out;
-    }
+    return g_bus_get_sync (G_BUS_TYPE_SYSTEM, cancellable, error);
```

Comment:
This is relatively unusual as far as our control flow goes; we don't have many early *successful* returns.  I think it's worth a comment here.